packages/client/ui/Select/index.tsx

```diff
@@ -0,0 +1,6 @@
+export {Select} from './Select'
+export {SelectContent} from './SelectContent'
+export {SelectGroup} from './SelectGroup'
+export {SelectItem} from './SelectItem'
+export {SelectTrigger} from './SelectTrigger'
+export {SelectValue} from './SelectValue'
```

Comment:
+1 IMO index files just mean 1 more piece of boilerplate to keep updated when we add/remove components, but if everyone likes the pattern I don't mind that much. The index AND additional file that just renames the radix export seem a bit much though. Maintaining 2 additional files that creates 3 sources of truth means someone could import the same thing 3 ways:

```ts
import {SelectValue} from '../ui/Select'
import {SelectValue} from '../ui/Select/SelectValue'
import {Value as SelectValue} from '@radix-ui/react-select'
```

IMO we should have a single source of truth & just import from radix, but I'm open to your opinion as well as @BartoszJarocki as we nail down the right radix-ui patterns for us.

if you wanted to reduce the 3 sources of truth to 2, you could also do

```ts
// index.tsx
export {Value as SelectValue} from '@radix-ui/react-select'
```

Reply:
no idea how I missed that file! I'm not w fan of index files as @mattkrick said, I think it's just another place to maintain.

I like the idea of reexporting/renaming radix component that's a part of a bigger solution, just because there's a pretty big chance we'll end up customizing it, as well as for naming consistency reason. If the component is only used internally by some kind of abstraction that we introduce, no need to rename/reexport it. that's just my 2c.

Reply:
@mattkrick For some reason I wanted to make imports less noisy and also thought about this `ui` folder as our UI library which potentially could be a separate package.

No strong preference here, I changed imports back to `import {SelectValue} from '../ui/Select/SelectValue'`

However I would agree with @BartoszJarocki and prefer to reexport some of the components because we may customize it later

Reply:
works for me!